### PR TITLE
Redesign

### PR DESCRIPTION
**Pull request by zutrinken**
### WIP 🚧

Should be deployed simultaneously with the app release.

**Things to work on:**

- [x] Show Android device and screenshot on Android phones
- [x] Update Screenshots

## Frontage
Bildschirmfoto 2019-04-10 um 11 59 28
## Map
Bildschirmfoto 2019-04-10 um 12 00 07
## Gallery
Bildschirmfoto 2019-04-10 um 12 00 17
## Info
Bildschirmfoto 2019-04-10 um 12 00 28

### CODE REVIEW

**Comment by headione** [comment]
So great! ❤️

**Comment by stephanlindauer** [comment]
:heart_eyes: wow!
i will take a closer look later today :wink:

File: src/css/_base.scss [hunk 1]
```diff
@@ -30,43 +30,43 @@ body {
...
   }
 
+  h1,
   h2 {
-    font-size: 1.75em;
-    line-height: 1.143em;
-    margin: 2.286em 0 -0.571em;
+    font-size: 32px;
+    line-height: 48px;
+    margin: 40px 0 24px;
```
[the remark this answers is not in the document]
Comment (zutrinken) [reply src/css/_base.scss]: yes. done ✅

File: src/css/_base.scss [hunk 2]
```diff
@@ -81,20 +81,17 @@ body {
   ul > li {
     &:before {
```
[the remark this answers is not in the document]
Comment (zutrinken) [reply src/css/_base.scss]: done ✅

File: src/css/_base.scss [hunk 3]
```diff
@@ -110,18 +107,17 @@ body {
 
   ol,
```
[the remark this answers is not in the document]
Comment (zutrinken) [reply src/css/_base.scss]: Not sure why classes are needed. It's supposed to be a base formatting.

**Pull request merged**